test/msw-api/setup-worker/start/wait-until-ready.error.test.ts

```diff
@@ -5,6 +5,8 @@ test('restores deferred requests if the worker registration fails', async () =>
   const { page } = await pageWith({
     example: path.resolve(__dirname, 'wait-until-ready.error.mocks.ts'),
     routes(app) {
+      app.set('etag', false)
```

Comment:
This prevents express from responding with 304 to the cached responses. Always force 200.